tests/core/engine_adapter/__init__.py

```diff
@@ -17,3 +17,9 @@ def to_sql_calls(adapter: EngineAdapter, identify: bool = True) -> t.List[str]:
         )
         output.append(sql)
     return output
+
+
+def temp_table_name(table_name: str, random_id: str):
```

Comment:
It looks like this is just used by the tests. If so, it would be better to have the function just exist in the tests then. You can make it be a callable fixture that can be passed in. You will see examples of this where we follow the pattern of `make_*`.